fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -36,9 +36,20 @@ set(FastRTPS_FOUND FALSE)
 find_path(FastRTPS_INCLUDE_DIR
   NAMES fastrtps/)
 
+find_library(FastRTPS_LIBRARY
+    NAMES fastrtps-1.4 fastrtps)
```

Comment:
Good catch! I've pushed https://github.com/ros2/rmw_fastrtps/pull/75/commits/011fc1401378ce27c336f2a0e73b1cea5d975879 which uses the `fastCDR_VERSION` and `fastRTPS_VERSION` variables from their CMake modules so any time they bump version, this would still work.